src/app/AttributeCache.cpp

```diff
@@ -219,5 +274,112 @@ CHIP_ERROR AttributeCache::GetStatus(const ConcreteAttributePath & path, StatusI
     return CHIP_NO_ERROR;
 }
 
+void AttributeCache::GetSortedFilters(std::vector<std::pair<DataVersionFilter, size_t>> & aVector)
```

Comment:
This function is still completely undocumented.

Reply:
just put comment in AttributeCache.h, thanks